CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## [0.17.0] - 2016-08-23
```

Comment:
You'll also need to add a line at the bottom of this file so the `[0.17.0]` link works.